tests/Form/Helper/FormHelperTest.php

```diff
@@ -13,18 +15,35 @@
 use Symfony\Component\Form\Extension\Core\Type\EmailType;
 use Symfony\Component\Form\Extension\Core\Type\IntegerType;
 use Symfony\Component\Form\Extension\Core\Type\NumberType;
+use Symfony\Component\Form\Extension\Core\Type\RangeType;
 use Symfony\Component\Form\Extension\Core\Type\TextType;
 use Symfony\Component\Form\Extension\Core\Type\TimeType;
 use Symfony\Component\Form\Extension\Core\Type\UrlType;
+use TypeError;
 
+/**
+ * Class FormHelperTest
+ * @package Cyve\JsonSchemaFormBundle\Tests\Form\Helper
+ * @coversDefaultClass \Cyve\JsonSchemaFormBundle\Form\Helper\FormHelper
+ */
 class FormHelperTest extends TestCase
 {
     /**
+     * @covers ::resolveFormType
      * @dataProvider resolveFormTypeDataProvider
+     * @param mixed $property
+     * @param mixed $expected
      */
-    public function testResolveFormType($schema, $expected)
+    public function testResolveFormType($property, $expected)
     {
-        $this->assertEquals($expected, FormHelper::resolveFormType($schema));
+        if (is_a($expected, LogicException::class, true)) {
```

Comment:
```suggestion
        if (is_a($expected, \LogicException::class, true)) {
```
And remove `use LogicException;` (and other native classes)
Testing for exceptions is a very good point, thx 👍